tasks/configure.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+
+- name: Get System fs.file-max
```

Comment:
I think the file limit settings should live in build.yml, configure is meant for environment variables and settings that need to be set when an instance boots up.

Reply:
Moved to build.yml